### ISSUE TITLE
Add names for profiles

### ISSUE THREAD

**Issue by nikidziuba** [symonxdd/epic-switcher-prototype#2]
Would it be possible to associate profile ids with names?
Would really clear up which account is which.

**Comment by symonxdd** [comment]
Thanks for leaving feedback!

This feature is already a work in progress! It seems though that it's rather not straightforward to have the app auto-assign profile name...
There's also a much clearer version almost ready, so if you'd like be sure to download the latest version then!

**Comment by psilynt1** [comment]
Just ran into this myself. It seems like you can simply rename the INI files that get created under AppData\Local\EpicGamesAccountSwitcher\AccountData to accomplish this.

**Comment by MCsanad** [comment]
Is it possible to edit account names manually?

**Comment by symonxdd** [comment]
@MCsanad No, unfortunately not atm. I'm working on my thesis presentation atm, but will do my best to implement this feature asap!

**Comment by MCsanad** [comment]
Good luck with your thesis!

**Comment by symonxdd** [comment]
Heyy! Hope you're vibing. Sooo my thesis went great and now I work at NASA… if we count daydreaming about space as employment 😭🌌
Manifesting though. Stay tuned.

So I’ve been working on an upgraded version of Epic Switcher that’s faster and improved overall. You can find it here: https://github.com/symonxdd/epic-switcher.

Would love to hear what you think if you get a chance to try it!
